tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/model/CalendarEventParse.scala

```diff
@@ -105,6 +106,24 @@ object CalendarFreeBusyStatusField extends Enumeration {
 
 case class CalendarFreeBusyStatusField(value: FreeBusyStatus) extends AnyVal
 
+object CalendarEventStatusField extends Enumeration {
+  type EventStatus = Value
+  val Confirmed = Value("confirmed")
+  val Cancelled = Value("cancelled")
+  val Tentative = Value("tentative")
```

Comment:
Shouldn't this be implemented as an enum? (sealed trait + case objects ?)

Reply:
Here I am using directly the enum syntax of Scala 2.xx: https://www.scala-lang.org/api/2.13.5/scala/Enumeration.html

Is it not less boilerplate than sealed trait + case objects?